.vib/airflow/cypress/cypress/support/commands.js

```diff
@@ -13,13 +13,13 @@ for (const command of ["click"]) {
 }
 
 Cypress.Commands.add(
-  "login",
-  (username = Cypress.env("username"), password = Cypress.env("password")) => {
+  'login',
+  (username = Cypress.env('username'), password = Cypress.env('password')) => {
     cy.clearCookies();
-    cy.visit("/login");
-    cy.get('form[name="login"]').should("exist");
-    cy.get("input#username").type(username);
-    cy.get("input#password").type(password);
+    cy.visit('/login');
+    cy.get('form[name="login"]').should('exist').and('be.visible');
```

Comment:
I've had a lot of trouble with the tests failing randomly because this form wasn't found. I empirically concluded that this assertion helps.

Reply:
Great! It's OK to resort to `be.visible` sometimes, but we should add a comment in the code.